docs/docs-content/clusters/edge/site-deployment/initial-setup.md

```diff
@@ -0,0 +1,52 @@
+---
+sidebar_label: "Initial Edge Host Configuration"
+title: "Initial Edge Host Configuration"
+description: "Instructions for configuring the Edge host when booting up the Edge host for the first time."
+hide_table_of_contents: false
+sidebar_position: 32
+tags: ["edge"]
+---
+
+When you boot up an Edge host for the first time after installation, you will be prompted to configure the Edge host and its network environment. You may already have specified some of these
+configurations in the **user-data** file in the EdgeForge process or have supplied them with site-specific
+**user-data**, and can either keep them as they are or update them during this step. For more information about
```

Comment:
@lennessyy maybe it's me but the sentence that starts with "You may already was a bit tricky to read.  It might be easier to read if we break into two sentences.